Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -103,6 +107,7 @@ open class MessageContentCell: MessageCollectionViewCell {
         cellBottomLabel.text = nil
         messageTopLabel.text = nil
         messageBottomLabel.text = nil
+        timeLabel.attributedText = nil
```

Comment:
I wonder why we reset `text` for the others if we set `attributedText` to them too?

Reply:
Exactly :)

Also, For performance reasons, you should only reset attributes of the cell that are not related to content, for example, alpha, editing, and selection state. [Reference](https://developer.apple.com/documentation/uikit/uitableviewcell/1623223-prepareforreuse)